tests/mock.js

```diff
@@ -1,24 +1,34 @@
 /*!
  * Copyright (c) 2018-2019 Digital Bazaar, Inc. All rights reserved.
  */
+'use strict';
```

Comment:
Don't need this.
```suggestion
```

Reply:
This isn't coming from an ESM module or is everything the tests require `esm` modules? If this results in a duplicate `use strict` I understand, but I thought the issue was the `esm` conversion the library uses results in `use strict` being there by default.

Reply:
I think for the purposes here, if code is using import/export then it is in module mode and strict by default.

Reply:
ok my apologies will remove them now.